torch/ao/pruning/_experimental/pruner/saliency_pruner.py

```diff
@@ -0,0 +1,20 @@
+from .base_structured_sparsifier import BaseStructuredSparsifier
+
+class SaliencyPruner(BaseStructuredSparsifier):
+     """
+     Prune filters based on the saliency
+     The saliency for a filter is given by the sum of the L1 norms of all of its weights
```

Comment:
might be helpful to document the dimension of the weight we are expecting (and add an assert) and over which dimension we are calculating L1 norma as well I think